packages/kubernetes-1.26/kubelet-exec-start-conf

```diff
@@ -2,10 +2,12 @@
 ExecStart=
 ExecStart=/usr/bin/kubelet \
 {{#unless settings.kubernetes.standalone-mode}}
-{{#if (eq settings.kubernetes.cloud-provider "")}}
-    --cloud-provider "" \
-{{else}}
+{{#if (eq settings.kubernetes.cloud-provider "external")}}
```

Comment:
Is there ever a case where the value of `settings.kubernetes.cloud-provider` should be used directly and not set to `"external"` in the generated output? If not, it would be simpler to do:

```txt
{{#if (ne settings.kubernetes.cloud-provider "")}}
    --cloud-provider "external" \
{{else}}
    --cloud-provider "" \
{{/if}}
```

See test here:
https://gist.github.com/stmcginnis/326a239daef49a84916bb96c85bba80c

Reply:
In that case, I believe you can just do this:
```
{{#if settings.kubernetes.cloud-provider }}
  --cloud-provider "external" \
{{else}}
   --cloud-provider "" \
{{/if}}
```

Reply:
I tried this way and failed on test. I was thinking if this will be true when we set `settings.kubernetes.cloud-provider` to "".